org-code-javabuilder/lib/src/main/java/org/code/javabuilder/Util.java

```diff
@@ -56,4 +63,53 @@ public static void recursivelyClearDirectory(Path directory) throws IOException
     Files.walk(directory).sorted(Comparator.reverseOrder()).map(Path::toFile).forEach(File::delete);
     LoggerUtils.sendClearedDirectoryLog(directory);
   }
+
+  public static void throwInvalidClassException(NoClassDefFoundError e)
```

Comment:
nit: could this be potentially a little more descriptive, like `convertAndThrowInvalidClassException`? that's a little long, so maybe not necessary, but I'll leave it up to you.

Reply:
done!